tests/TestHooks.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace TestsNextGen;
+
+/**
+ * @unreleased
+ *
+ * TODO: move into GiveWP Core
+ *
+ */
+class TestHooks
```

Comment:
What's the purpose of this?

Reply:
It's a convenient facade i'm working on for using WordPress test suite hooks.  You can see it in action in `tests/bootstrap.php`.  It's main purpose it to simplify the bootstrap process for add-ons so testing out here and will eventually move into core.

Updated with more descriptions f17a2320